test/edge.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { writeRscDataEdgeFunction } from '../packages/runtime/src/helpers/edge'
+import type { PrerenderManifest } from 'next/dist/build'
+
+const basePrerenderManifest: PrerenderManifest = {
+  version: 3,
+  routes: {},
+  dynamicRoutes: {},
+  notFoundRoutes: [],
+  preview: {
+    previewModeId: '',
+    previewModeSigningKey: '',
+    previewModeEncryptionKey: '',
+  },
+}
+
+describe('writeRscDataEdgeFunction', () => {
```

Comment:
Sorry github won't let me comment in the other file as you didn't edit this part. Can we rename this function (`writeRscDataEdgeFunction`)? It doesn't actually doing the "write" in the function, it just returns an array.

Reply:
sure thing, fixed in https://github.com/netlify/next-runtime/pull/2044/commits/1141f4ff6b2296f2d9fb85056beed29eb10cfe7e